app/lib/cluster_viz_service.rb

```diff
@@ -13,7 +13,7 @@ def self.get_cluster_group(study, params)
 
   # helper method to load all possible cluster groups for a study
   def self.load_cluster_group_options(study)
-    non_spatial_file_ids = StudyFile.where(study: study, :is_spatial.ne => true, file_type: 'Cluster').pluck(:id)
+    non_spatial_file_ids = study.clustering_files.where(is_spatial: false).pluck(:id)
```

Comment:
Nice catch!  Are queries involving `file_type: <Foo>` prevalent?  If so, I can imagine we'd need similar updates in a fair number of other places to account for AnnData files.

Reply:
Great call - according to my IDE (assuming I got the regex right) there are ~20 occurrences of `StudyFile.where` that include `file_type` as a parameter.  I'll go ahead and review each one to determine if we need to amend the queries.

Reply:
[0bd7122](https://github.com/broadinstitute/single_cell_portal_core/pull/1703/commits/0bd71220f6c0254dfe785e0ea7bba31baa91c539)